src/renderer/vsync/vsync_win_dwm.rs

```diff
@@ -94,7 +90,7 @@ impl VSyncWinDwm {
                     tracy_plot!("sleep_time", _sleep_time);
 
                     if redraw_requested.swap(false, Ordering::Relaxed) {
-                        let _ = proxy.send_event(UserEvent::RedrawRequested);
+                        proxy.send_event(UserEvent::RedrawRequested).unwrap();
```

Comment:
We don't want to crash here, if the proxy is gone, but it can be changed to 
`proxy.send_event(UserEvent::RedrawRequested).ok()`, That format seem to be more common in our code than `let _ =`